TuneUp/TuneUp.csproj

```diff
@@ -118,7 +118,7 @@
     <Copy SourceFiles="@(PackageJson)" DestinationFolder="$(PackageFolder)" />
     <MakeDir Directories="$(ExtraFolder)" Condition="!Exists($(ExtraFolder))"/>
     <MakeDir Directories="$(DyfFolder)" Condition="!Exists($(DyfFolder))"/>
-    <CallTarget Condition="'$(Configuration)' == 'Debug'" Targets="PackageDeploy" />
+    <CallTarget Condition="'$(Configuration)' == 'Release'" Targets="PackageDeploy" />
```

Comment:
Forgot to clarify.. I don't think we want to commit this one, the change here is for publisher only